Classes_Homework5/exec_utils/configloader.py

```diff
@@ -0,0 +1,13 @@
+from configparser import ConfigParser, ExtendedInterpolation
+import os
+
+
+class Config:
+    configParser = ConfigParser(interpolation=ExtendedInterpolation())
+    config_file_path = '\\'.join(os.path.realpath(__file__).split('\\')[0:-1]) + r'\configs.ini'  # identifies the same directory as this module.
```

Comment:
I'm not sure that that will work on Linux or Ubuntu machines because `\\` is a windows separator.

You can use **os.sep** which gives the OS-dependent path separator (/ for Linux/Unix, \\ for Windows)

![image](https://user-images.githubusercontent.com/34323554/109434890-28facf80-7a20-11eb-87c8-59f787e322ca.png)